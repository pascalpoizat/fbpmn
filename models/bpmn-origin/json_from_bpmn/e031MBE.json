{
    "isInterrupt": {
        "BoundaryEvent_1g6jly9": true
    },
    "catN": {
        "Task_0dvwezg": "SendTask",
        "ExclusiveGateway_1uyyba4": "AndGateway",
        "ExclusiveGateway_04stbki": "XorGateway",
        "StartEvent_1": "NoneStartEvent",
        "BoundaryEvent_1g6jly9": "MessageBoundaryEvent",
        "EndEvent_07lweo2": "NoneEndEvent",
        "EndEvent_03g8tis": "NoneEndEvent",
        "StartEvent_0pevvbh": "NoneStartEvent",
        "Task_05h2dnk": "AbstractTask",
        "ExclusiveGateway_06yyw6a": "AndGateway",
        "P_": "Process",
        "Task_1rjwiou": "AbstractTask",
        "SubProcess_0ikactq": "SubProcess",
        "Task_1kf2bvv": "AbstractTask",
        "StartEvent_1em4yhi": "NoneStartEvent",
        "Q_": "Process",
        "EndEvent_1bmsqno": "NoneEndEvent"
    },
    "catE": {
        "SequenceFlow_1uzescv": "NormalSequenceFlow",
        "SequenceFlow_13u1qxz": "NormalSequenceFlow",
        "SequenceFlow_1t3nh6y": "NormalSequenceFlow",
        "MessageFlow_1arf9uv": "MessageFlow",
        "SequenceFlow_0aq5v2x": "NormalSequenceFlow",
        "SequenceFlow_0fh3j7y": "NormalSequenceFlow",
        "SequenceFlow_1wro3pq": "NormalSequenceFlow",
        "SequenceFlow_07f8qba": "NormalSequenceFlow",
        "SequenceFlow_1qymta9": "NormalSequenceFlow",
        "SequenceFlow_0ylruuw": "NormalSequenceFlow",
        "SequenceFlow_1gn3q83": "NormalSequenceFlow",
        "SequenceFlow_0j6nnaa": "NormalSequenceFlow",
        "SequenceFlow_0r3bens": "NormalSequenceFlow",
        "SequenceFlow_1ofsg25": "NormalSequenceFlow"
    },
    "containN": {
        "P_": [
            "StartEvent_1",
            "ExclusiveGateway_1uyyba4",
            "Task_1rjwiou",
            "Task_05h2dnk",
            "ExclusiveGateway_04stbki",
            "SubProcess_0ikactq",
            "ExclusiveGateway_06yyw6a",
            "EndEvent_1bmsqno",
            "BoundaryEvent_1g6jly9"
        ],
        "SubProcess_0ikactq": [
            "StartEvent_1em4yhi",
            "Task_1kf2bvv",
            "EndEvent_07lweo2"
        ],
        "Q_": [
            "StartEvent_0pevvbh",
            "EndEvent_03g8tis",
            "Task_0dvwezg"
        ]
    },
    "messageE": {
        "MessageFlow_1arf9uv": "m1"
    },
    "timeInformation": {},
    "targetE": {
        "SequenceFlow_1uzescv": "ExclusiveGateway_1uyyba4",
        "SequenceFlow_13u1qxz": "EndEvent_1bmsqno",
        "SequenceFlow_1t3nh6y": "Task_05h2dnk",
        "MessageFlow_1arf9uv": "BoundaryEvent_1g6jly9",
        "SequenceFlow_0aq5v2x": "ExclusiveGateway_04stbki",
        "SequenceFlow_0fh3j7y": "ExclusiveGateway_06yyw6a",
        "SequenceFlow_1wro3pq": "ExclusiveGateway_06yyw6a",
        "SequenceFlow_07f8qba": "ExclusiveGateway_04stbki",
        "SequenceFlow_1qymta9": "Task_1rjwiou",
        "SequenceFlow_0ylruuw": "SubProcess_0ikactq",
        "SequenceFlow_1gn3q83": "EndEvent_03g8tis",
        "SequenceFlow_0j6nnaa": "EndEvent_07lweo2",
        "SequenceFlow_0r3bens": "Task_0dvwezg",
        "SequenceFlow_1ofsg25": "Task_1kf2bvv"
    },
    "name": "e031MBE",
    "attached": {
        "BoundaryEvent_1g6jly9": "SubProcess_0ikactq"
    },
    "containE": {
        "P_": [
            "SequenceFlow_1uzescv",
            "SequenceFlow_1qymta9",
            "SequenceFlow_1t3nh6y",
            "SequenceFlow_07f8qba",
            "SequenceFlow_0aq5v2x",
            "SequenceFlow_0ylruuw",
            "SequenceFlow_13u1qxz",
            "SequenceFlow_1wro3pq",
            "SequenceFlow_0fh3j7y"
        ],
        "SubProcess_0ikactq": [
            "SequenceFlow_1ofsg25",
            "SequenceFlow_0j6nnaa"
        ],
        "Q_": [
            "SequenceFlow_0r3bens",
            "SequenceFlow_1gn3q83"
        ]
    },
    "edges": [
        "MessageFlow_1arf9uv",
        "SequenceFlow_1uzescv",
        "SequenceFlow_1qymta9",
        "SequenceFlow_1t3nh6y",
        "SequenceFlow_07f8qba",
        "SequenceFlow_0aq5v2x",
        "SequenceFlow_0ylruuw",
        "SequenceFlow_13u1qxz",
        "SequenceFlow_1wro3pq",
        "SequenceFlow_0fh3j7y",
        "SequenceFlow_1ofsg25",
        "SequenceFlow_0j6nnaa",
        "SequenceFlow_0r3bens",
        "SequenceFlow_1gn3q83"
    ],
    "messages": [
        "m1"
    ],
    "sourceE": {
        "SequenceFlow_1uzescv": "StartEvent_1",
        "SequenceFlow_13u1qxz": "ExclusiveGateway_06yyw6a",
        "SequenceFlow_1t3nh6y": "ExclusiveGateway_1uyyba4",
        "MessageFlow_1arf9uv": "Task_0dvwezg",
        "SequenceFlow_0aq5v2x": "Task_05h2dnk",
        "SequenceFlow_0fh3j7y": "BoundaryEvent_1g6jly9",
        "SequenceFlow_1wro3pq": "SubProcess_0ikactq",
        "SequenceFlow_07f8qba": "Task_1rjwiou",
        "SequenceFlow_1qymta9": "ExclusiveGateway_1uyyba4",
        "SequenceFlow_0ylruuw": "ExclusiveGateway_04stbki",
        "SequenceFlow_1gn3q83": "Task_0dvwezg",
        "SequenceFlow_0j6nnaa": "Task_1kf2bvv",
        "SequenceFlow_0r3bens": "StartEvent_0pevvbh",
        "SequenceFlow_1ofsg25": "StartEvent_1em4yhi"
    },
    "nodes": [
        "P_",
        "Q_",
        "StartEvent_1",
        "ExclusiveGateway_1uyyba4",
        "Task_1rjwiou",
        "Task_05h2dnk",
        "ExclusiveGateway_04stbki",
        "SubProcess_0ikactq",
        "ExclusiveGateway_06yyw6a",
        "EndEvent_1bmsqno",
        "BoundaryEvent_1g6jly9",
        "StartEvent_1em4yhi",
        "Task_1kf2bvv",
        "EndEvent_07lweo2",
        "StartEvent_0pevvbh",
        "EndEvent_03g8tis",
        "Task_0dvwezg"
    ],
    "nameN": {
        "Task_0dvwezg": "D",
        "Task_05h2dnk": "B",
        "Task_1rjwiou": "A",
        "SubProcess_0ikactq": "SP",
        "Task_1kf2bvv": "C"
    }
}