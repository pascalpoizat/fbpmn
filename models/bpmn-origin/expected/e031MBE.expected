Working in /tmp/e031MBE.Kc2Du with 1 worker(s)
[92mtransformation done[0m
<<"Processes=", 2, "Nodes=", 17, "Gateway=", 3, "SF=", 13, "MF=", 1>>
---------- Network01Bag ----------
[X] Prop01Type
     states=360 trans=922 depth=25
[ ] Prop02Safe
     states= trans= depth=9
[ ] Prop03Sound
     states=360 trans=922 depth=
[ ] Prop04MsgSound
     states=360 trans=922 depth=
---------- Network02FifoPair ----------
[X] Prop01Type
     states=360 trans=922 depth=25
[ ] Prop02Safe
     states= trans= depth=9
[ ] Prop03Sound
     states=360 trans=922 depth=
[ ] Prop04MsgSound
     states=360 trans=922 depth=
---------- Network03Causal ----------
[X] Prop01Type
     states=360 trans=922 depth=25
[ ] Prop02Safe
     states= trans= depth=9
[ ] Prop03Sound
     states=360 trans=922 depth=
[ ] Prop04MsgSound
     states=360 trans=922 depth=
---------- Network04Inbox ----------
[X] Prop01Type
     states=360 trans=922 depth=25
[ ] Prop02Safe
     states= trans= depth=9
[ ] Prop03Sound
     states=360 trans=922 depth=
[ ] Prop04MsgSound
     states=360 trans=922 depth=
---------- Network05Outbox ----------
[X] Prop01Type
     states=360 trans=922 depth=25
[ ] Prop02Safe
     states= trans= depth=9
[ ] Prop03Sound
     states=360 trans=922 depth=
[ ] Prop04MsgSound
     states=360 trans=922 depth=
---------- Network06Fifo ----------
[X] Prop01Type
     states=360 trans=922 depth=25
[ ] Prop02Safe
     states= trans= depth=9
[ ] Prop03Sound
     states=360 trans=922 depth=
[ ] Prop04MsgSound
     states=360 trans=922 depth=
---------- Network07RSC ----------
[X] Prop01Type
     states=360 trans=922 depth=25
[ ] Prop02Safe
     states= trans= depth=9
[ ] Prop03Sound
     states=360 trans=922 depth=
[ ] Prop04MsgSound
     states=360 trans=922 depth=
done.
